tools/HttpExceptions/GenerateClasses.php

```diff
@@ -19,11 +19,11 @@ abstract class Code {
     }
 
     $namespace = "Tnapf\Router\Exceptions";
-    
+
     $parsedPhrase = preg_replace("/[^a-zA-Z]/", "", $code->phrase);
     $className = "Http".$parsedPhrase;
 
-    ob_start(); 
+    ob_start();
     echo "<?php\n"; ?>
```

Comment:
But I need this so the code is PHP

Reply:
Can you explain? I've just removed excess spaces here.
The only notable change I've made is the `define()` to `const` conversion, which does the exact same thing actually, but it's the new standard.

Reply:
It looks like you removed echo <?php

Reply:
Oh wait, nevermind. Seems github mobile is just being stupid